generator/gm/gm_global_a_powertrain.dbc

```diff
@@ -0,0 +1,2 @@
+CM_ "IMPORT _gm_global_a_powertrain.dbc";
+CM_ "IMPORT _comma.dbc";
```

Comment:
just import the comma DBC into the GM one. no need for an intermediate GM one

Reply:
I didn't see any examples like that - Should I put the import at the end of the file, beginning or somewhere in the middle?